pkg/helper/features/features.go

```diff
@@ -0,0 +1,38 @@
+package features
+
+import (
+	"fmt"
+
+	"github.com/openshift/rosa/pkg/helper/versions"
+)
+
+type OCMFeature string
+
+const (
+	AdditionalDay2SecurityGroupsHcpFeature = "AdditionalDay2SecurityGroupsHcpFeature"
+)
+
+var ocmFeatureVersions = map[OCMFeature]string{
+	AdditionalDay2SecurityGroupsHcpFeature: "4.15.0-0.a",
+}
+
+func IsFeatureSupported(feature OCMFeature, version string) (bool, error) {
+	// The version is not specified.
+	if version == "" {
+		return true, nil
+	}
+
+	supportedVersion := ocmFeatureVersions[feature]
+	// OCM feature isn't defined.
+	if supportedVersion == "" {
+		return true, nil
+	}
```

Comment:
As this is a public helper function, we should add a comment that clearly describes what the function does. The comment should include a statement that if the version string provided is empty or if the feature is not defined, the function will return `true` nonetheless.

Reply:
Added, thanks.